internal/cli/atlas_alert_config_create.go

```diff
@@ -0,0 +1,236 @@
+	}
+
+	if opts.metricThresholdMetricName != "" {
+		out.MetricThreshold = newMetricThreshold(opts)
+	}
+
+	out.Notifications = []atlas.Notification{*newNotification(opts)}
+
+	return out
+}
+
+func newNotification(opts *atlasAlertConfigCreateOpts) *atlas.Notification {
+
+	out := new(atlas.Notification)
+	out.TypeName = strings.ToUpper(opts.notificationType)
+	out.DelayMin = &opts.notificationDelayMin
+	out.IntervalMin = opts.notificationIntervalMin
+	out.TeamID = opts.notificationTeamID
+	out.Username = opts.notificationUsername
+
+	switch out.TypeName {
+
+	case victor:
+		out.VictorOpsAPIKey = opts.apiKey
+		out.VictorOpsRoutingKey = opts.notificationVictorOpsRoutingKey
+
+	case slack:
+		out.VictorOpsAPIKey = opts.apiKey
+		out.VictorOpsRoutingKey = opts.notificationVictorOpsRoutingKey
+		out.APIToken = opts.notificationToken
+
+	case datadog:
+		out.DatadogAPIKey = opts.apiKey
+		out.DatadogRegion = strings.ToUpper(opts.notificationRegion)
+
+	case email:
+		out.EmailAddress = opts.notificationEmailAddress
+
+	case flowdock:
+		out.FlowdockAPIToken = opts.notificationToken
+		out.FlowName = opts.notificationFlowName
+		out.OrgName = opts.notificationOrgName
+
+	case sms:
+		out.MobileNumber = opts.notificationMobileNumber
+
+	case group, user, org:
+		out.SMSEnabled = &opts.notificationSmsEnabled
+		out.EmailEnabled = &opts.notificationEmailEnabled
+
+	case ops:
+		out.OpsGenieAPIKey = opts.apiKey
+		out.OpsGenieRegion = opts.notificationRegion
+
+	case pager:
+		out.ServiceKey = opts.notificationServiceKey
+
+	}
+
+	return out
+}
+
+func newMetricThreshold(opts *atlasAlertConfigCreateOpts) *atlas.MetricThreshold {
+	return &atlas.MetricThreshold{
+		MetricName: strings.ToUpper(opts.metricThresholdMetricName),
+		Operator:   strings.ToUpper(opts.metricThresholdOperator),
+		Threshold:  opts.metricThresholdThreshold,
+		Units:      strings.ToUpper(opts.metricThresholdUnits),
+		Mode:       strings.ToUpper(opts.metricThresholdMode),
+	}
+}
+
+func newMatcher(opts *atlasAlertConfigCreateOpts) *atlas.Matcher {
+	return &atlas.Matcher{
+		FieldName: strings.ToUpper(opts.matcherFieldName),
+		Operator:  strings.ToUpper(opts.matcherOperator),
+		Value:     strings.ToUpper(opts.matcherValue),
+	}
+}
+
+// mcli atlas alert-config(s) create -event event --enabled [--matcherField fieldName --matcherOperator operator --matcherValue value]
+// [--notificationType type --notificationDelayMin min --notificationEmailEnabled --notificationSmsEnabled --notificationUsername username --notificationTeamID id
+// --notificationEmailAddress email --notificationMobileNumber number --notificationChannelName channel --notificationApiToken --notificationRegion region] [--projectId projectId]
+func AtlasAlertConfigCreateBuilder() *cobra.Command {
+	opts := &atlasAlertConfigCreateOpts{
+		globalOpts: newGlobalOpts(),
+	}
+	cmd := &cobra.Command{
+		Use:     "create",
+		Short:   "Create an Atlas alert configuration for a project.",
+		Aliases: []string{"cr", "Create"},
```

Comment:
Is `cr` a common alias for cli? I don't think I've seen it before but happy to be proven wrong, also we don't offer alias with caps for other commands so we can safety remove `Create`